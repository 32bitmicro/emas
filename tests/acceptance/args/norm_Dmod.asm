lw r1, [r2]
lw r1, [r2+r3]
lw r1, [3]
lw r1, [r4+1]
lw r1, [1+r5]
lw r1, [r6-1]
