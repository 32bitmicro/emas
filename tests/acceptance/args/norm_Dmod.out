@ 0x0000 : 0x4242  /  010 000 1 001 000 010  /  16962
@ 0x0001 : 0x425a  /  010 000 1 001 011 010  /  16986
@ 0x0002 : 0x4240  /  010 000 1 001 000 000  /  16960
@ 0x0003 : 0x0003  /  000 000 0 000 000 011  /  3
@ 0x0004 : 0x4260  /  010 000 1 001 100 000  /  16992
@ 0x0005 : 0x0001  /  000 000 0 000 000 001  /  1
@ 0x0006 : 0x4268  /  010 000 1 001 101 000  /  17000
@ 0x0007 : 0x0001  /  000 000 0 000 000 001  /  1
@ 0x0008 : 0x4270  /  010 000 1 001 110 000  /  17008
@ 0x0009 : 0xffff  /  111 111 1 111 111 111  /  -1
